dashboard/src/setupTests.ts

```diff
@@ -2,7 +2,6 @@ import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
 import Enzyme from "enzyme";
 import "jest-enzyme";
 import { WebSocket } from "mock-socket";
-import "raf/polyfill"; // polyfill for requestAnimationFrame
```

Comment:
Odd that it was only used in tests for a polyfill. Guess browsers handle this properly now. https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame#browser_compatibility - seems fully supported.